crates/compiler/builtins/bitcode/src/dec.zig

```diff
@@ -656,6 +656,9 @@ fn div_u256_by_u128(numer: U256, denom: u128) U256 {
     // 1 <= sr <= N_UTWORD_BITS - 1
     var carry: u128 = 0;
 
+    const i128_max: i128 = 170141183460469231731687303715884105727;
```

Comment:
Can you use [std.math.maxInt](https://ziglang.org/documentation/master/std/#A;std:math.maxInt) instead. Same with min?